runck

```diff
@@ -77,6 +87,8 @@ launch() {
     chmod 0755 "${tmpfile}"
   fi
 
+  echo ""
```

Comment:
I thought we were outputing that to stderr, but I see we don't. We should output stuff to stderr.